### PR TITLE
[Docs] 3rd Party BGP support by Cilium in EKS-A

### PR DESCRIPTION
**Pull request by amitmavgupta**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by amitmavgupta** [comment]
@chrisdoherty4 @chrisnegus @csplinter

File: docs/content/en/docs/packages/metallb/addmetallb.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ Be sure to refer to the [troubleshooting guide]({{< relref "../troubleshoot" >}}
   {{% alert title="Important" color="warning" %}}
    * Starting at `eksctl anywhere` version `v0.12.0`, packages on workload clusters are remotely managed by the management cluster.
    * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be run with `kubeconfig` pointing to the workload cluster.
+   * Operators can also integrate with Isovalent's for 3rd party BGP support. Isovalent can integrate with:
```
Comment (chrisnegus) [inline docs/content/en/docs/packages/metallb/addmetallb.md:16]: Do you mean this?: Operators can also integrate with Isovalent's 3rd party BGP support. Or is there something called "Isovalent's for 3rd party BGP support" ?

**Review by chrisnegus — COMMENTED** [review]
One small comment.

File: docs/content/en/docs/packages/metallb/addmetallb.md (hunk 1 above)
Comment (csplinter) [reply docs/content/en/docs/packages/metallb/addmetallb.md:16]: We should not have this on the MetalLB page, especially considering it looks like Cilium is going to remove their integration with MetalLB https://github.com/cilium/cilium/issues/22246#issuecomment-1381207686

**Pull request closed without merging**